service/r2/routes_test.go

```diff
@@ -198,9 +333,70 @@ func newTestInstrumentMethodMetrics() instrument.MethodMetrics {
 	return instrument.NewMethodMetrics(tally.NoopScope, "testRoute", 1.0)
 }
 
+func newBulkReqBody() updateRuleSetRequest {
```

Comment:
Please add `test` to the method names in tests to distinguish them from normal non-test code, e.g., `newTestBulkReqBody`.

Reply:
Ping